website/docs/components/icon/partials/code/how-to-use.md

```diff
@@ -124,25 +124,25 @@ The [Hds::Button](/components/button?tab=code#icon-only-button) component automa
 The default value is `currentColor` which uses the inherited text color as the icon color. When setting a custom value, we recommend using one of the pre-defined variables to ensure consistency with our design language:
 
 ```handlebars
-<FlightIcon @name="zap" @color="var(--brand)" />
+<Hds::Icon @name="zap" @color="var(--brand)" />
 ```
 
 Other accepted values include named colors and color values themselves (e.g., hex, rgb, etc).
 
 ```handlebars
-<FlightIcon @name="zap" @color="rebeccapurple" />
+<Hds::Icon @name="zap" @color="rebeccapurple" />
 ```
 
 ```handlebars
-<FlightIcon @name="zap" @color="rgb(46, 113, 229)" />
+<Hds::Icon @name="zap" @color="rgb(46, 113, 229)" />
 ```
 
 ### Size
 
 The default size is 16px. To use the alternative 24px icon size, set the `@size` value:
 
 ```handlebars
-<FlightIcon @name="zap" @size="24" />
+<Hds::Icon @name="zap" @size="24" />
 ```
 
 ### Stretched
```

Comment:
In the paragraph below, there is a reference to `(width: 100%, height: 100%, display: block)`, we can remove the `display: block` from it.